packages/now-node/src/dev-server.ts

```diff
@@ -52,23 +52,72 @@ async function main() {
     config.helpers === false || process.env.NODEJS_HELPERS === '0'
   );
 
-  const entrypointPath = path.join(process.cwd(), entrypoint);
-  const handler = await import(entrypointPath);
+  bridge = getNowLauncher({
```

Comment:
Does this also support the AWS Lambda API `getAWSLauncher()`?

Reply:
Not yet, it'll have to be a follow-up PR. There's already so much changing in this one.